packages/syft/src/syft/client/client.py

```diff
@@ -514,6 +512,7 @@ def register(
             return SyftError(message=str(e))
         response = self.connection.register(new_user=new_user)
         if isinstance(response, tuple):
+            self.credentials = response[1].signing_key
```

Comment:
I think there was a feature request,to prevent register method from modifying the user.
I think Koen made a PR earlier to remove the same.
Do we want to have the same user?

Reply:
👍🏾